### PR TITLE
build and enable selinux

### PR DESCRIPTION
**Pull request by cooktheryan**
Completes the integration of Selinux

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -183,6 +182,14 @@ User=root
 WantedBy=multi-user.target
 EOF
 
+    sudo mkdir -p /var/run/flannel
+    sudo mkdir -p /var/run/kubelet
+    sudo mkdir -p /var/lib/kubelet/pods
+    sudo mkdir -p /var/run/secrets/kubernetes.io/serviceaccount
+    sudo mkdir -p /var/hpvolumes
+    sudo semodule -i /tmp/microshift.pp
```
Comment (rootfs) [inline install.sh:190]: will /tmp survive?

Comment (cooktheryan) [reply install.sh:190]: Once the pp file is loaded it is no longer needed. We could potentially add in a delete of the file once loaded

Comment (rootfs) [reply install.sh:190]: as long as it is persisted, it is good

**Comment by cooktheryan** [comment]
@rootfs i dont know if this needs manually merged.

**Comment by rootfs** [comment]
@cooktheryan manual it is :D

**Pull request merged**